spec/lib/miq_automation_engine/miq_ae_service_spec.rb

```diff
@@ -31,6 +31,14 @@ module MiqAeServiceSpec
         @object.attributes.should == original_attributes
       end
     end
+
+    context "#inspect" do
+      it "returns the class, id and name" do
+        @object.stub(:object_name).and_return('fred')
+        str = @service_object.inspect
+        /#<MiqAeMethodService::MiqAeServiceObject:0x(\w+) name:.\"(?<name>\w+)\">/ =~ str
```

Comment:
This is weird to have a =~ but not use the results directly.

Maybe prefer something like

```ruby
match = /the regex/.match(str)
match[:name].should eq('fred')
```